src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/IntegerItem.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\Plugin\Field\FieldType\IntegerItem as IntegerItemExtend;
+
+/**
+ * {@inheritdoc}
+ */
+class IntegerItem extends IntegerItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $values = parent::generateSampleValue($field_definition);
+    switch ($field_definition->getName()) {
+      // Random values must be unique.
```

Comment:
due to some `unique` database constraint?

Reply:
Correct, the year is the identifier for the `annual-report`. I'm not sure if the value needs to be unique to pass the validation. But, it should be unique.